ui/pages/confirmation/templates/add-ethereum-chain.js

```diff
@@ -146,21 +146,28 @@ const ERROR_CONNECTING_TO_RPC = {
   },
 };
 
+async function fetchSafeChainsList() {
+  try {
+    const response = await fetchWithCache({
+      url: 'https://chainid.network/chains.json',
+      // cacheOptions: { cacheRefreshTime: DAY },
```

Comment:
Needs removal (?)

Reply:
Removed during this refactor PR, after removing the usage in getAlerts
https://github.com/MetaMask/metamask-extension/pull/22085